module/cpf.sh

```diff
@@ -26,17 +29,22 @@ declare -A COLORS=(
 )
 
 #. cpf:module -={
-function ::cpf:module_is_modified() {
+function ::cpf:module_is_unchanged() {
```

Comment:
That it confused me. It returned 0 when it was not modified, which I interpret as true. I just believed the use of it is clearer if you see the function name as a question. I hope that makes sense.